### DISCUSSION TITLE
ModV on Mac help (Big Sur)

### DISCUSSION THREAD

**Discussion by SFDxST** [vcync/modV#677] [Q&A] [answered]
Hey everyone. So I recently downloaded the latest build of ModV and after unzipping the file, I tried to launch the program. To my surprise, my Macbook is telling me that I do not "have permission" to launch ModV. 

I'm obviously doing something wrong here and I humbly ask for your guidance

**Comment by 2xAA** [comment, marked as answer]
Hey @SFDxST - looks like the app in the ZIP file is missing permissions, we'll raise an issue to look into it.

Have you tried the DMG instead? Seems to work.
Latest (at time of posting) here: https://github.com/vcync/modV/releases/download/3.20.1/modV-3.20.1.dmg
